dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/RumFeatureTest.kt

```diff
@@ -217,9 +220,17 @@ internal class RumFeatureTest : SdkFeatureTest<Any, Configuration.Feature.RUM, R
     }
 
     @Test
-    fun `𝕄 setup vital monitors 𝕎 initialize()`() {
+    fun `𝕄 setup vital monitors 𝕎 initialize { frequency != NEVER }`(forge: Forge) {
```

Comment:
```
    @ParameterizedTest
    @EnumSource(VitalsUpdateFrequency::class, names = ["NEVER"], mode = EnumSource.Mode.EXCLUDE)
    fun `𝕄 setup vital monitors 𝕎 initialize { frequency != NEVER }`(fakeFrequency: VitalsUpdateFrequency) {
        …
    )
```

Reply:
Yeah...make sense to have a parameterized test here instead.